Airports/EGAC/Runway.txt

```diff
@@ -1 +1 @@
-22  04  220 040 N054.37.29.340 W005.51.52.140 N054.36.43.260 W005.52.47.240
\ No newline at end of file
+04  22  040 220 N054.36.43.260 W005.52.47.240 N054.37.29.340 W005.51.52.140
```

Comment:
```suggestion
04  22  037 217 N054.36.43.340 W005.52.47.140 N054.37.29.480 W005.51.51.950
```
This one seems to be off for some reason, please do check my numbers.

Reply:
Is it the only one or have I really messed something up here

Reply:
It's the only one I spotted.

Reply:
Ok, probably one of the ones i had to do manually haha